plugins/inspect/os_inspector.rb

```diff
@@ -96,8 +101,8 @@ def get_os_from_os_release(system)
   end
 
   # checks for old suse standard: /etc/SuSE-release
-  def get_os_from_suse_release(system)
-    suse_release = system.read_file("/etc/SuSE-release")
+  def get_os_from_suse_release
```

Comment:
Assignment Branch Condition size for get_os_from_suse_release is too high. [26.94/15]